migrations/sql/V2021.08.20.16.09__create_itrb_exemption_code_type_codes.sql

```diff
@@ -0,0 +1,35 @@
+CREATE TABLE IF NOT EXISTS itrb_exemption_status (
+    itrb_exemption_status_code       varchar                                NOT NULL PRIMARY KEY,
+    description                      varchar                                NOT NULL            ,
+    active_ind                       boolean                  DEFAULT true  NOT NULL            ,
+    create_user                      varchar                                NOT NULL            ,
+    display_order					 smallint                               NOT NULL            ,
+    create_timestamp                 timestamp with time zone DEFAULT now() NOT NULL            ,
+    update_user                      varchar                                NOT NULL            ,
+    update_timestamp                 timestamp with time zone DEFAULT now() NOT NULL
+);
+
+
+ALTER TABLE itrb_exemption_status OWNER TO mds;
+
+INSERT INTO itrb_exemption_status
+    (
+    itrb_exemption_status_code,
+    description,
+    display_order,
+    create_user,
+    update_user
+    )
+VALUES
+    ('NO', 'No', 10, 'system-mds', 'system-mds'),
+    ('YES', 'Yes', 20, 'system-mds', 'system-mds'),
+    ('EXEM', 'Exemption', 30, 'system-mds', 'system-mds')
+ON CONFLICT DO NOTHING;
+
```

Comment:
I briefly chatted with Aaron after the meeting to add a bit more clarity to these labels. 
can you update the descriptions to the following: 

```('NO', 'No - Without Exemption', 10, 'system-mds', 'system-mds'),
 ('YES', 'Yes', 20, 'system-mds', 'system-mds'),
 ('EXEM', 'No - With Exemption', 30, 'system-mds', 'system-mds')```